gaming/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
nit: This file does not seem to do anything. Or am I missing something here?

Reply:
Yeah, it was a placeholder. I'm going to need a script eventually to init the sample app for later work, but it looks like it will be python, not bash. So I'll remove this file from the PR.